codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Connection.java

```diff
@@ -43,8 +43,10 @@
         void onStreamActive(Http2Stream stream);
 
         /**
-         * Notifies the listener that the given stream is now {@code HALF CLOSED}. The stream can be
-         * inspected to determine which side is {@code CLOSED}.
+         * Notifies the listener that the given stream has transitioned from {@code OPEN} to {@code HALF CLOSED}.
+         * This method will <strong>not</strong> be called until a state transition occurs from when
+         * {@link #onStreamActive(Http2Stream)} was called.
+         * The stream can be inspected to determine which side is {@code HALF CLOSED}.
```

Comment:
@nmittler - Note I changed from `{@code CLOSED}` to `{@code HALF CLOSED}`...I think this makes more sense. WDYT?